tests/UnitBLAS/Level2.hs

```diff
@@ -7,7 +7,6 @@
 
 module UnitBLAS.Level2(unitTestLevel2BLAS) where 
 
-import Test.HUnit
 --import Numerical.Array.Shape as S 
```

Comment:
In my work environment, the "@?=" mark is ambiguous for Test.HUnit and Test.Tasty.HUnit. I found the easiest way to make it compilable is to delete the Test.HUnit.
Sorry for forgot to add it back.
What is the correct approach to solve the ambiguous problem?